src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/SearchPageObject.java

```diff
@@ -27,7 +27,8 @@ public SearchPageObject(WebDriver driver) {
   protected WebElement paginatorNext;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)